2022-07-30 13:36:41,835 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-07-30 13:36:41,835 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\diedi\AppData\Local\Programs\Python\Python39\lib\site-packages\instabot\bot\bot.py
2022-07-30 13:36:41,836 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-07-30 13:36:42,156 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-07-30 13:36:42,156 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-07-30 13:36:42,162 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-07-30 13:36:42,163 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2022-07-30 13:41:42,463 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-07-30 13:41:42,463 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-07-30 13:41:42,463 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-07-30 13:41:42,463 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 10 minutes.
2022-07-30 13:51:42,686 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [400]>
2022-07-30 13:51:42,686 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"","status":"fail"}'
2022-07-30 13:51:42,686 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-07-30 13:51:42,687 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: 
2022-07-30 13:51:42,687 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2022-07-30 13:51:42,951 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-07-30 13:51:43,211 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-07-30 13:51:43,411 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-07-30 13:51:51,159 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-07-30 13:51:51,159 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https:\\/\\/i.instagram.com\\/challenge\\/AXH_C1aeUNIQ6wJp3SBli3mm0MGYnWwvAkDmTCdnZCcBg-F3wfARupdXZUOAcwv3vsiiDDY\\/AfxMXzsFwgU4J3Y5JilJ9gMJwcd8huRB5Uy3O1qbrmbLiYm_9kFmkpM3chcBWN3CqsQ5t8dFMOEVKw\\/","api_path":"\\/challenge\\/AXH_C1aeUNIQ6wJp3SBli3mm0MGYnWwvAkDmTCdnZCcBg-F3wfARupdXZUOAcwv3vsiiDDY\\/AfxMXzsFwgU4J3Y5JilJ9gMJwcd8huRB5Uy3O1qbrmbLiYm_9kFmkpM3chcBWN3CqsQ5t8dFMOEVKw\\/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required","exception_name":"ChallengeRequired"}'
2022-07-30 13:51:51,160 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-07-30 13:51:51,161 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-07-30 13:51:51,161 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
2022-07-30 13:51:51,162 - instabot version: 0.117.0 (api) - CRITICAL - Not logged in!
